### ISSUE TITLE
App degradation on slow network

### ISSUE THREAD

**Issue by serkanatpaystack** [ooade/react-starwars#2]
When I select a movie, change the selection and get back to the initial movie, all characters seem to be loading again from the network.

And if in between, my connection drops, I don't get to see the characters even though they have been loaded before.

Is there a way to eliminate the need to reload the characters over and over?

**Comment by serkanatpaystack** [comment]
So, @ooade do you think you can create a PR for this?

**Comment by ooade** [comment]
Yes, there's a way to elimate that actually, using memoization. I'll send a PR for this.
